src/styles/components/app-header.styl

```diff
@@ -0,0 +1,55 @@
+.app-header
+  display: flex
+  align-items: flex-end
+  margin: $unit-size (3 * $unit-size)
+  font-size: 14px
+  border-bottom: 1px solid $grey
+  
+  .app-title
+    color: $text-color
+    
+    img  //To be replaced with the proper Liberating the Liberator logo
+      display: inline-block
+      height: 1em
+      vertical-align: middle
+    h1  //To be rendered invisible, or removed altogether
+      display: inline-block
+      font-size: 1em
+      vertical-align: middle
+
+  nav
+    flex: 1 1 auto
+    display: flex
+    justify-content: flex-end
+    align-items: center
```

Comment:
You might need some sort of wrap around here, because the navbar looks quite awkward in mobile view.